### ISSUE TITLE
 Request help "Error:feature_calculator"

### ISSUE THREAD

**Issue by parez93** [kyamagu/paperdoll#2]
I need some help for running paperdoll (pre-trained parser for a new image).

When I run:
load data/paperdoll_pipeline.mat config;
input_image = imread('./my_image.jpg');
input_sample = struct('image', imencode(input_image, 'jpg'));
result = feature_calculator.apply(config, input_sample); %error

The result comes:
```
[06-Nov-2016 06:29:48] Error:feature_calculator: 1 / 1
Attempted to access pose_struct(1,:); index out of bounds because size(pose_struct)=[0,0].

Error in normal_pose_calculator.apply (line 40)
      pose_struct = pose.PARSE_from_UCI(pose.box2point(pose_struct(1,:)));

Error in feature_calculator.apply (line 25)
        sample = calculators{j}(config{j}, sample, varargin{:}, 'Encode', false);


result =

     []
```

Is there any solution to fix it ?

While platforms are:
- ubuntu 12.04 64 bit
- Matlab R2012a
- opencv 2.4.11
- no error when i run makefile

**Comment by kyamagu** [comment]
@parez93 You'll get this when pose estimator fail to find a person. Try lowering the pose threshold as indicated in the README.

**Comment by parez93** [comment]
Thank you very much!
I've another question:
when i run 
`show_parsing(result.image, result.final_labeling, result.refined_labels);` 
I get this error:
```
[`Error using horzcat
Dimensions of matrices being concatenated are not consistent.

Error in show_parsing>reorder_labels (line 23)
  ordered_labels = [reserved_labels, setdiff(labels, reserved_labels)];

Error in show_parsing (line 12)
  [labeling, labels] = reorder_labels(uint8(labeling), labels);
```
Maybe it is a my mistake, but if i change in "show_parsing.m" function, all go right:
`  ordered_labels = [reserved_labels, setdiff(labels, reserved_labels)];
`
in 
`  ordered_labels = [reserved_labels, setdiff(labels, reserved_labels)'];
`
(i added a single quote at the end).

**Comment by kyamagu** [comment]
@parez93 Right, [this commit](https://github.com/kyamagu/paperdoll/commit/d10035dd4655142820b2e7cd85f3417af9ce07dd) should have fixed this.
